Arelle/Arelle
# Package Windows zip build

#### Reason for change
resolves #481 

#### Description of change
* update the Windows workflow to produce a zip build.
* replace duplicated paths/filenames with env vars.
* fail workflows if build artifacts are missing and can't be uploaded.

#### Steps to Test
* smoke test the Windows builds (installer and zip) from the [testing release on my fork](https://github.com/austinmatherne-wk/Arelle/releases/tag/2.2.11).
* verify that the [updated macOS build](https://github.com/Arelle/Arelle/actions/runs/3453636182) still produces a DMG using the path env var.

**review**:
@Arelle/arelle
